src/main/java/core/basesyntax/service/action/BalanceAction.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.action;
+
+import core.basesyntax.model.Fruit;
+
+public class BalanceAction implements ActionHandler {
+
+    @Override
+    public void count(Fruit fruit, int amount) {
+        fruit.setBalance(amount);
```

Comment:
work with Storage map here 
```suggestion
        Storage.storage.put(fruit, amount);
```
Fix other handlers